presto-native-execution/scripts/setup-centos.sh

```diff
@@ -67,7 +67,7 @@ export COMPILER_FLAGS=$(echo -n $(get_cxx_flags $CPU_TARGET))
   git clone https://github.com/facebook/folly &&
   cd folly &&
   git checkout $FB_OS_VERSION &&
-  cmake_install -DBUILD_TESTS=OFF -DBUILD_SHARED_LIBS=ON
```

Comment:
FYI, I have landed this already :)

Reply:
> FYI, I have landed this already :)

Ya, you PR https://github.com/prestodb/presto/pull/19866 changes setup-macos.sh but updating docker image requires setup-centos.sh as well.